src/spec.ts

```diff
@@ -420,6 +420,7 @@ function normalizeOverlay(spec: UnitSpec, overlayWithPoint: boolean, overlayWith
         filled: true,
         role: 'pointOverlay'
       },
+      ...(selection ? {selection} : {}),
```

Comment:
say you have colored multi-series line chart  and selection is projected over color.